docs/apps/lnw/es2k/es2k-linux-networking.md

```diff
@@ -24,11 +24,11 @@ This topology breakdown and configuration assumes all VMs are spawned on HOST VF
 
 ### Topology breakdown
 
-- Every VM spawned on top of a VF will have a corresponding port representer in ACC.
-- Every physical port will have a corresponding port representer in ACC.
-- Every physical port will have an uplink (APF netdev) in HOST and this uplink will have a corresponding port representer in ACC.
-- All port representers are associated with an OvS bridge.
-- For VxLAN egress traffic, the underlay port should be associated with a termination bridge and IP to reach the underlay network should be configured on top of this bridge.
+- Every VM spawned on top of a VF will have a corresponding port representor in ACC.
```

Comment:
I was leaning towards `representor` since Linux kernel documentation and DPDK documentation online uses `representor` for similar concept.
https://docs.kernel.org/networking/representors.html
https://doc.dpdk.org/guides-18.05/prog_guide/switch_representation.html#port-representors